kpops/_version.py

```diff
@@ -0,0 +1 @@
+__version__ = "1.1.1"
```

Comment:
@disrupted We have to keep in mind that this file needs to be updated whenever we do a release.

Reply:
yeah, I would prefer updating it with a Poetry plugin during version bumping, rather than doing it in the CI release pipeline.

Reply:
Not sure what release process is followed, but the idea was to let python-poetry-bump-version action handle that (ref: https://github.com/bakdata/ci-templates/pull/95)

Reply:
If we use either the [poetry-bumpversion](https://github.com/monim67/poetry-bumpversion) plugin, or read it dynamically using `importlib` we can entirely skip the additional CI step.

```python
__version__= importlib.metadata.version("kpops")
```

Reply:
also please move this to `kpops/__init__.py` and use an absolute import `from kpops import __version__`

Reply:
> also please move this to `kpops/__init__.py` and use an absolute import `from kpops import __version__`

great suggestions, applied